src/main/lombok/org/heigit/ohsome/ohsomeapi/output/Description.java

```diff
@@ -3,118 +3,152 @@
     }
-    return "Total " + label + " of items in " + unit + ", aggregated on the boundary.";
   }
 
   public static String aggregateGroupByBoundaryGroupByTag(boolean isDensity, String label,
       String unit) {
     if (isDensity) {
-      return "Density of selected items (" + label + " of items in " + unit + " divided by the "
-          + "area in square kilometers), aggregated on the boundary and on the tag.";
+      return String.format(DENSITY_OF_ITEMS_IN_UNIT, label, unit)
+          + AGGREGATED_ON_THE_BOUNDARY_AND_TAG;
+    } else {
+      return String.format(NUMBER_OF_ITEMS_IN_UNIT, label, unit)
+          + AGGREGATED_ON_THE_BOUNDARY_AND_TAG;
     }
-    return "Total " + label + " of items in " + unit
-        + ", aggregated on the boundary and on the tag.";
   }
 
   public static String aggregateGroupByUser(String label, String unit) {
-    return "Total " + label + " of items in " + unit + ", aggregated on the user.";
+    return "Total " + label + " of items in " + unit + AGGREGATED_ON_THE_USER;
   }
 
   public static String aggregateGroupByTag(boolean isDensity, String label, String unit) {
     if (isDensity) {
-      return "Density of selected items (" + label + " of items in " + unit
-          + " divided by the area in square kilometers), aggregated on the tag.";
+      return String.format(DENSITY_OF_ITEMS_IN_UNIT, label, unit) + AGGREGATED_ON_THE_TAG;
+    } else {
+      return String.format(NUMBER_OF_ITEMS_IN_UNIT, label, unit) + AGGREGATED_ON_THE_TAG;
     }
-    return "Total " + label + " of items in " + unit + ", aggregated on the tag.";
   }
 
   public static String countPerimeterAreaGroupByType(boolean isDensity, String label, String unit) {
     if (isDensity) {
-      return "Density of selected items (" + label + " of items in " + unit
-          + " divided by the area in square kilometers), aggregated on the type.";
+      return String.format(DENSITY_OF_ITEMS_IN_UNIT, label, unit) + AGGREGATED_ON_THE_TYPE;
+    } else {
+      return String.format(NUMBER_OF_ITEMS_IN_UNIT, label, unit) + AGGREGATED_ON_THE_TYPE;
     }
-    return "Total " + label + " of items in " + unit + ", aggregated on the type.";
   }
 
   public static String aggregateGroupByKey(String label, String unit) {
-    return "Total " + label + " of items in " + unit + ", aggregated on the key.";
+    return String.format(NUMBER_OF_ITEMS_IN_UNIT, label, unit) + AGGREGATED_ON_THE_KEY;
   }
 
   public static String aggregateRatio(String label, String unit) {
-    return "Total " + label + " of items in " + unit
+    return String.format(NUMBER_OF_ITEMS_IN_UNIT, label, unit)
         + " satisfying types2, keys2, values2 parameters (= value2 output),"
-        + " as well as items selected by types, keys, values parameters (= value output) "
-        + "and ratio of value2:value.";
+        + " as well as items selected by types, keys, values parameters (= value output)"
+        + " and ratio of value2:value.";
```

Comment:
this should be a static variable as well, as it is used in `aggregateRatioGroupByBoundary(String, String)` as well